### DISCUSSION TITLE
New Features or Miss Items

### DISCUSSION THREAD

**Discussion by FrostBreker** [FrostBreker/OneTwentyBackport-1.19.3#2] [Ideas]
Please let me know if I miss something or if you have any suggestions !

**Comment by CapeSword** [comment]
I'm having trouble designing a modpack with the Trails & Tales Backport in 1.19.2 as it disables mining picks. When I try to mine any stone, I can't. I tested it with stone tools as well and it still didn't work. I isolated the Trails and Tales from the other mods and couldn't get the problem resolved on my own. When I remove the Trails & Tales Backport, and use the One Twenty Backport instead, the problem stops, but I don't like using that one as your backport is more feature-rich. Can you help with this?

**Reply by FrostBreker** [reply]
Fix in next update V0.1.1 for 1.19.2
#4

**Reply by CapeSword** [reply]
Thank you

**Reply by MohamedOsamaYA** [reply]
add 1.16.5 for the mod

**Comment by tizu69** [comment]
Technical changes, mainly the display entities and interaction entity. Also, if possible, the /execute changes?

**Reply by FrostBreker** [reply]
Currently commands and technical things are not my priority ^^

**Comment by redkittty** [comment]
There was a new structure added called the Trail Ruins

**Reply by FrostBreker** [reply]
Added in the tasks !

**Comment by lenase0077** [comment]
Biome spawn?

**Reply by FrostBreker** [reply]
Added in the tasks !

**Comment by Nude-Zenpai** [comment]
> Please let me know if I miss something or if you have any suggestions !

new soundtracks/maybe discs? , new ruined structures for the archeology stuff and the pots , suspicious sand

**Reply by FrostBreker** [reply]
Soundtracks and Discs are not my priority currently ^^ but the archelogy will come !

**Comment by Hipposgrumm** [comment]
I know I made an issue for this (since I didn't see the discussion), but I backported armor trimming already and was hoping you could recognize that.

**Reply by FrostBreker** [reply]
Hi, I'm working on it ^^

**Comment by DraconicArcher** [comment]
There is an error in the loot table of cherry_leaves where it reads minecraft:cherry_leaves instead of onetwenty:cherry_leaves. This errors out the loot table, making cherry saplings unobtainable in survival.

**Reply by FrostBreker** [reply]
In which version, I will check it

**Reply by DraconicArcher** [reply]
Forge 1.18.2  onetwenty-0.3-1.18.2

**Reply by FrostBreker** [reply]
Fix!

https://legacy.curseforge.com/minecraft/mc-mods/onetwenty-1-20-backport/files/4613500

**Comment by MohamedOsamaYA** [comment]
> Please let me know if I miss something or if you have any suggestions !

CAN YOU ADD 1.16.5 Ur Work IS VERY COOOOOL!

**Comment by MohamedOsamaYA** [comment]
I am waited so much

**Comment by MohamedOsamaYA** [comment]
> Please let me know if I miss something or if you have any suggestions !

PLEASE ADD 1.16.5 THATS WHAT I WANTED

**Comment by Fool1k** [comment]
Can you please add the Torch flower and Pitcher plant? They're the two plants that the Sniffer digs up.

**Comment by anaischirinos** [comment]
> Please let me know if I miss something or if you have any suggestions !
Hi wondering if you will be adding the cherry blossom pink petals??

**Reply by GEORGO-HTML100500** [reply]
I really need an auto-crafter, please add it!
